encoding/protobuf/cue/cue.proto

```diff
@@ -17,5 +22,5 @@ message FieldOptions {
 
 extend google.protobuf.FieldOptions {
     string val = 123456;
-    FieldOptions opt = 1069;
+    FieldOptions opt = 1070;
```

Comment:
what is this used for? (why the need to increment?)

Reply:
I can likely undo this. It was based on https://developers.google.com/protocol-buffers/docs/proto#updating. I was concerned that since we were updating the package, we might need to update the field number for consistency. This field isn't being used like other fields (for marshalling) so we don't have to be as strict with non-breaking changes.

Reply:
Undid the change